t/extract-package.t

```diff
@@ -127,6 +127,36 @@ $VERSION = '999';
 $::VERSION = 0.01;
 ---
 },
+{
+  name => 'class NAME::SUBNAME',
+  package => [ 'Simple::Edward' ],
+  code => <<'---',
+class Simple::Edward;
+---
+},
+{
+  name => 'class NAME::SUBNAME extends NAME',
+  package => [ 'Simple::Edward' ],
+  code => <<'---',
+class Simple::Edward extends Another::Package;
```

Comment:
The `extends` keyword is an old one from `Object::Pad`, that isn't part of the core perl syntax. There's no need to test this one.